tests/Shared/LayoutRenderers/AspNetSessionValueLayoutRendererTests2.cs

```diff
@@ -49,19 +49,6 @@ public void MissingItemRendersEmpty()
             // Assert
             Assert.Empty(result);
         }
-        [Fact]
-        public void ThrowItemRendersEmpty()
```

Comment:
Because the is invalid. LayoutRenderers are allowed to throw exceptions. Layouts are not allowed to throw exceptions. This PR removes an exception-catch-all, which should never have been there.